components/eventCard/EventCard.tsx

```diff
@@ -108,62 +102,44 @@ const styles = StyleSheet.create({
     flexDirection: "row"
   },
   topRow: {
-    paddingBottom: 16,
+    paddingBottom: 12,
     alignItems: "center"
   },
   middleRow: {
     flex: 1,
     flexDirection: "column"
   },
-  location: {
-    paddingBottom: 8
+  bottomSpacing: {
+    marginBottom: 8
   },
   infoText: {
     textAlignVertical: "center",
+    fontSize: 14,
+    fontFamily: "Open-Sans",
     color: "grey"
   },
-  attendingNumber: {
-    fontWeight: "bold"
-  },
-  attendingText: {
-    textAlignVertical: "center",
-    fontSize: 14
-  },
   titleText: {
     textAlignVertical: "center",
-    fontWeight: "bold",
-    fontSize: 22,
-    marginBottom: 8
+    fontFamily: "Open-Sans-Bold",
+    fontSize: 16
   },
   name: {
     textAlignVertical: "center",
-    fontWeight: "bold",
-    fontSize: 15,
+    fontFamily: "Open-Sans-Bold",
+    fontSize: 14,
     paddingLeft: 8
   },
-  distance: {
-    flexDirection: "row",
-    alignSelf: "center",
-    paddingVertical: 2,
-    paddingLeft: 4,
-    borderRadius: 12
-  },
-  distanceContainer: {
-    flexDirection: "row"
-  },
-  distanceText: {
-    textAlignVertical: "center",
-    color: "white",
-    paddingRight: 8,
-    fontWeight: "bold"
+  description: {
+    fontFamily: "Open-Sans",
+    fontSize: 13
   },
   image: {
-    width: 40,
-    height: 40,
+    width: 32,
+    height: 32,
     borderRadius: 24
   },
   container: {
-    backgroundColor: "white",
+    backgroundColor: "#f5f5f5",
```

Comment:
We should have this in a separate file so it's easy to reuse in other components